examples/applications/plot_model_complexity_influence.py

```diff
@@ -162,8 +191,43 @@ def _count_nonzero_coefficients(estimator):
      'prediction_performance_label': 'MSE',
      'n_samples': 30},
 ]
+
+
+##############################################################################
+# Run the code and plot the results
+# ---------------------------------
+# We are ready to run all our functions by looping though the configurations we
+# set previously and then plotting the influence of varying parameters on the
+# complexity and the latency of each model.
+#
+
+
+def plot_influence(conf, mse_values, prediction_times, complexities):
```

Comment:
I assume that we have a twin axes and on the left y axis, we report the prediction error (we can make the number blue as well) while on the right axis, we don't have any number. I think that we need to have the number with red

Something similar to: https://matplotlib.org/3.1.1/gallery/ticks_and_spines/multiple_yaxis_with_spines.html#sphx-glr-gallery-ticks-and-spines-multiple-yaxis-with-spines-py

Since we are changing the plot, we can use `tab:blue` and `tab:orange` instead of the red and blue.